src/sonic-daemon-common/sonic_daemon_common/daemon_common.py

```diff
@@ -0,0 +1,125 @@
+#!/usr/bin/env python2
+
+try:
+    import imp
+    import signal
+    import subprocess
+    import sys
+    import syslog
+except ImportError, e:
+    raise ImportError (str(e) + " - required module not found")
+
+#============================= Constants =============================
+
+# Platform root directory inside docker
+PLATFORM_ROOT_DOCKER = "/usr/share/sonic/platform"
+SONIC_CFGGEN_PATH = '/usr/local/bin/sonic-cfggen'
+HWSKU_KEY = 'DEVICE_METADATA.localhost.hwsku'
+PLATFORM_KEY = 'DEVICE_METADATA.localhost.platform'
+
+class daemon_util(object):
```

Comment:
Suggest rename class to "DaemonBase". With this change, I guess the package should probably also be renamed to "sonic_daemon_base".

Reply:
Agree.